osu.Game/Localisation/DiscordRichPresenceModeStrings.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class DiscordRichPresenceModeStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.DiscordRichPresenceMode";
+
+        /// <summary>
+        /// "Hide identifiable information"
+        /// </summary>
+        public static LocalisableString HideIdentifiableInformation => new TranslatableString(getKey(@"hide_identifiable_information"), @"Hide identifiable information");
+
+        /// <summary>
+        /// "Full"
+        /// </summary>
+        public static LocalisableString Full => new TranslatableString(getKey(@"rich_presence.full"), @"Full");
```

Comment:
What are the prefixes inside of `getKey` here?

Reply:
I could've seen something like that in other string classes, or just misread something, either way, there should not be any prefixes, as getKey already adds one.